### DISCUSSION TITLE
Manpager gives only on very few posisitons color

### DISCUSSION THREAD

**Discussion by frere-jacques** [sharkdp/bat#2814] [Q&A]
bat 0.24.0 (fc954685)
I am using arch with terminal foot and zsh. I tried it too with bash, but with same results.
I have this in my .zshrc
export MANPAGER="sh -c 'col -bx | bat -l man -p'"
export MANROFFOPT="-c"

The second helped to get rid of weird characters. But I get only on very few words colors.
I really don't see what the problem is. Looking online it should show much more.

Edit: I figured out it is related to the theme. I have set theme to ansi, because I wanted bat to just follow my terminal colors. If I unset the envvar BAT_THEME. I have a lot more of colors in the man output. So I guess it means I shouldn't use ansi, but find a proper theme with my desired colors instead.

If there is another workaround I am happy to hear about it.

Edit2: Found a fitting theme https://github.com/mhanberg/everforest-textmate/blob/main/Everforest%20Dark/Everforest%20Dark.tmTheme
But using that theme gives same result. Only a few words at all colored.
I guess it's related to #2115

**Comment by frere-jacques** [comment]
![20231228Dez121703795196](https://github.com/sharkdp/bat/assets/26696299/fa101ce5-ef28-425e-b0eb-6e0e30ed0283)
Here is a picture.

**Comment by keith-hall** [comment]
[the Manpage syntax is currently looking for specific headings](https://github.com/sharkdp/bat/blob/5e77ca37e89c873e4490b42ff556370dc5c6ba4f/assets/syntaxes/02_Extra/Manpage.sublime-syntax#L46) in order to assign colors to that section. Probably we need to add support for your language (German)?

**Reply by frere-jacques** [reply]
Interesting, did not thought it might be related to localization.

**Reply by keith-hall** [reply]
To be fair, you're probably hitting the theme issue too at the same time. But PRs to improve localized Manpage highlighting will be gratefully accepted. (I don't think there is a viable alternative to get correct coloring regardless of language locale due to how arbitrary man pages are)

**Reply by frere-jacques** [reply]
I forgot completely on the topic.
Short question. I actually don't like to have my complete locale on english. But I am totaly fine with english man pages.
I use the following export to use bat as manpager:
```
export MANPAGER="sh -c 'col -bx | bat --theme=default -l man -Len -p'"
```
I read that one can overwrite the locale for man via -L and other methods. Is there a rather uncomplicated way to modify this export to use english man pages?

**Reply by frere-jacques** [reply]
Okay, I think this has nothing to do with the locale.
I started a terminal and run:
`export LANG=en_US.utf8`
and then `man man`.
When I don't overwrite the theme in the MANPAGER export, it looks in english nearly the same. See english output:
![20240608Jun061717848231](https://github.com/sharkdp/bat/assets/26696299/24219d42-c828-47c3-9c3a-804ae8558800)
